backend/src/data/repositories/user/user.repository.ts

```diff
@@ -11,8 +13,11 @@ class User {
     this.#UserModel = UserModel;
   }
 
-  async getAll(): Promise<UserM[]> {
-    return this.#UserModel.query();
+  async getAll({ page, count }: UserGetAllRequestQueryDto): Promise<UserM[]> {
```

Comment:
it's not getAll anymore

Reply:
@PavelSukhinin should I rename it or have both `getAll` and `getAllPaginated` ?

Reply:
I believe we won't need to get all users ever, so you can rename it